src/main/java/com/project/service/UserServiceImpl.java

```diff
@@ -79,7 +78,7 @@ public User updateUser(User user, long id) throws UserException {
 
     @Override
     public void deleteUser(long id) throws UserNotFoundException {
-        if(!userRepository.findById(id).isPresent())
+        if (!userRepository.findById(id).isPresent())
```

Comment:
`if` statements should be opened and closed with curly braces `{}` even if they have only one line of code.

Reply:
added curly braces `{}`  to all `if` statements in UserServiceImpl and ApplicationServiceImpl classes